### PR TITLE
add thai translation

### PR DESCRIPTION
**Pull request by wattachai**
I would like to add Thai translation to justdelete.me. Please review my PR. :)

### CODE REVIEW

File: _trans/th.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"extensionp3": "ต้องการติดตั้งส่วนเสริม? กรุณาไปที่นี่ <a target=\"_blank\" href=\"https://chrome.google.com/webstore/detail/justdeleteme/hfpofkfbabpbbmchmiekfnlcgaedbgcf\">Chrome Web Store</a>.",
+	"extensionl1": "<span class=\"dot-wrapper\"><span class=\"dot easy\"></span></span> - ขั้นตอนง่ายๆ",
+	"extensionl2": "<span class=\"dot-wrapper\"><span class=\"dot medium\"></span></span> - ต้องออกแรงนิดหน่อย",
+	"extensionl3": "<span class=\"dot-wrapper\"><span class=\"dot hard\"></span></span> - ไม่สามารถลบได้โดยไม่ติดต่อฝ่ายบริการลูกค้า",
+	"extensionl4": "<span class=\"dot-wrapper\"><span class=\"dot impossible\"></span></span> - ไม่สามารถลบได้",
+	"banner": "ลอง<span>ส่วนเสริมGoogle Chrome</span>ตัวใหม่ของเราสิ",
+	"footer": "สร้างสรรค์โดย <a href=\"http://robblewis.me\">Robb Lewis</a> และ <a href=\"http://edpoole.me\">Ed Poole</a> | Forkบน<a href=\"http://github.com/rmlewisuk/justdelete.me\">GitHub</a> | hostบน<a href=\"https://www.digitalocean.com/?refcode=8e1d8283bd20b\">DigitalOcean</a> | วิเคราะห์ข้อมูลด้วย<a href=\"https://t.co/fvPnva7p4Y\">GoSquared</a>",
```
Comment (tupaschoal) [inline _trans/th.json]: Please  remove everything after "| hostบ", including it, as this is not relevant anymore.

Change the fork link to https://github.com/jdm-contrib/justdelete.me

File: _trans/th.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	"difficulty_medium": "ปานกลาง",
+	"difficulty_hard": "ยาก",
+	"difficulty_impossible": "เป็นไปไม่ได้",
+	"tagline": "รายการdirect linkสำหรับลบบัญชีจากโลกออนไลน์์",
+	"noinfo": "ไม่มีข้อมูล",
+	"showinfo": "แสดงข้อมูล...",
+	"hideinfo": "ซ่อนข้อมูล...",
+	"whatisthis": "นี่คืออะไร?",
+	"whatisthis1": "หลายบริษัทใช้เทคนิค <a href=\"http://darkpatterns.org/\">dark pattern</a> เพื่อไม่ให้ผู้ใช้สามารถหาวิธีลบบัญชีออกได้โดยง่าย JustDelete.me ต้องการจะสร้างรายการurlเพื่อให้คุณสามารถลบบัญชีเหล่านี้ได้อย่างง่ายดาย",
+	"whatisthis2": "มีวิธีการลบบัญชีที่อยากจะเพิ่มเติม? <a href=\"http://github.com/rmlewisuk/justdelete.me\">ForkโครงการจากGitHubสิ</a>.",
```
Comment (tupaschoal) [inline _trans/th.json]: Change the link to https://github.com/jdm-contrib/justdelete.me

**Comment by wattachai** [comment]
Fixed as you suggest. Please have a look. 😄

**Comment by tupaschoal** [comment]
Looks fine to me nw @wattachai , thanks!

**Pull request merged**